packages/project-service/src/createProjectService.ts

```diff
@@ -3,27 +3,24 @@ import type * as ts from 'typescript/lib/tsserverlibrary';
 
 import debug from 'debug';
 
-import type { ProjectServiceOptions } from '../parser-options';
+import type { ProjectServiceOptions } from '@typescript-eslint/types';
```

Comment:
I initially didn't move ProjectServiceOptions here too, because other interfaces and types refer to it. But I wonder if types could take a dependency on this package....

Reply:
Ah, but then `types` would have a dependency on production code. Not great.